ui_espace_pro/src/common/utils/dateUtils.js

```diff
@@ -2,6 +2,17 @@ import dayjs from "dayjs"
 import relativeTime from "dayjs/plugin/relativeTime"
 import { NBR_EXPIRATION_J7 } from "../contants"
 
+export const formatDate = (date) => dayjs(date).format("YYYY-MM-DD")
+
+export const getCurrentDate = (separator = "/") => {
+  return dayjs(new Date()).format(`YYYY${separator}MM${separator}DD`)
+}
```

Comment:
Use dayjs + add jsdoc

Reply:
@rimiti there is already dayjs on this line. Ok for jsdocs